packages/teleport/src/Welcome/Welcome.tsx

```diff
@@ -21,13 +21,24 @@ import history from 'teleport/services/history';
 import LogoHero from 'teleport/components/LogoHero';
 import cfg from 'teleport/config';
 
+import { userEventService } from 'teleport/services/userEvent';
+
+import userEvents from 'teleport/services/userEvent/UserEvents/userEvents';
+
+import useToken from 'teleport/Welcome/useToken';
+
 import { NewCredentials } from './NewCredentials';
 import { CardWelcome } from './CardWelcome';
 
 export default function Welcome() {
   const { tokenId } = useParams<{ tokenId: string }>();
+  const { resetToken } = useToken(tokenId);
```

Comment:
not sure about this, the `useToken` fetches token data from the back, and doing this will fetch it here only to get the username, and fetch it again when user click "next" (form screen, line 70 and 73)

if the get started event is important, perhaps we can on [fetch token on init](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/Welcome/useToken.ts#L38), also send the capture event? This fetch on init happens in [NewCredentials](https://github.com/gravitational/webapps/blob/5b3a2dfa63faf2e11872a2805b9eabaac0731ffa/packages/teleport/src/Welcome/NewCredentials/NewCredentials.tsx) component

if it's important to capture the user event like "click" (ensure user performs an actual action) before we send an event, maybe we can send a state with the route similar to something we do here: https://github.com/gravitational/webapps/blob/master/packages/teleport/src/components/AgentButtonAdd/AgentButtonAdd.tsx#L45